handlers/whatsapp/whatsapp.go

```diff
@@ -473,6 +475,14 @@ func (h *handler) SendMsg(ctx context.Context, msg courier.Msg) (courier.MsgStat
 				if attachmentCount == 0 {
 					mediaPayload.Caption = msg.Text()
 				}
+				mediaPayload.Filename, err = utils.BasePathForURL(mediaURL)
+
+				logrus.WithField("channel_uuid", msg.Channel().UUID().String()).WithError(err).Error("Error while parsing the media URL")
+
+				// Break out on error
+				if err != nil {
+					break
+				}
```

Comment:
I think here you can create a channel log instead for the error and let the error to break on https://github.com/nyaruka/courier/pull/356/files#diff-557153ae44649c16db29083d386ef0de7914adfabb7e118a3b4e019b62a6e48bR525
after we save the logs so users and use can get a chance to see those logs on the web